src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -5833,6 +5830,18 @@ public void setCachedReplicaRowsFailThreshold(int threshold)
         logger.info("updated replica_filtering_protection.cached_rows_fail_threshold to {}", threshold);
     }
 
+    public int getColumnIndexSizeInKB()
+    {
+        return DatabaseDescriptor.getColumnIndexSizeInKB();
+    }
+
+    public void setColumnIndexSize(int columnIndexSizeInKB)
+    {
+        int oldValueInKB = DatabaseDescriptor.getColumnIndexSizeInKB();
+        DatabaseDescriptor.setColumnIndexSize(columnIndexSizeInKB);
+        logger.info("setColumnIndexSize: column index size set to {} KiB (was {} KiB)", columnIndexSizeInKB, oldValueInKB);
```

Comment:
How about "Updated column_index_size_in_kb to ..."? The other methods log the message in this format.